app/src/main/java/com/example/drawingactivity/DrawActivity.kt

```diff
@@ -32,8 +37,20 @@ class DrawActivity : AppCompatActivity() {
         setContentView(R.layout.activity_main)
         val drawableArea = findViewById<DrawableAreaView>(R.id.drawableArea)
         drawableArea.onBitmapDrawn = { bitmap -> viewModel.setCurrentBitmapState(bitmap) }
-        val colorSelector = findViewById<PrimaryColorCircleView>(R.id.colorCircle)
+        val colorSelector = findViewById<CurrentColorView>(R.id.colorCircle)
         colorSelector.onChangeColor = { viewModel.cycleCurrentColor() }
+        val coordinator = findViewById<CoordinatorLayout>(R.id.coordinator)
+        val bottomSheet = coordinator.findViewById<LinearLayout>(R.id.toolLayout)
+        val gradientTab = bottomSheet.findViewById<CurrentColorView>(R.id.pickerButton)
+        val brushTab = bottomSheet.findViewById<ImageView>(R.id.brushButton)
+        val moveTab = bottomSheet.findViewById<ImageView>(R.id.moveButton)
+        val resizeTab = bottomSheet.findViewById<ImageView>(R.id.resizeButton)
+        val filterTab = bottomSheet.findViewById<ImageView>(R.id.filterButton)
+        val layerTab = bottomSheet.findViewById<ImageView>(R.id.layerButton)
```

Comment:
Do we really _need_ to store all of these references? Maybe viewbinding will make this cleaner

Reply:
I stopped saving the references since they weren't being accessed outside of a forEach block, and just put the find view by ID in there instead.

I didn't touch viewbinding for this ticket and wanted to refactor it later, but I agree that viewbinding will make this a lot cleaner in the long run.